tools/packaging/packaging.psm1

```diff
@@ -665,6 +713,27 @@ function New-UnixPackage {
                 $newPackagePath = Join-Path $createdPackage.DirectoryName $newPackageName
                 $createdPackage = Rename-Item $createdPackage.FullName $newPackagePath -PassThru -Force:$Force
             }
+            if($pscmdlet.ShouldProcess("Cleanup macOS launcher"))
```

Comment:
Should this cleanup code be moved to the `if ($Environment.IsMacOS)` block in the `finally` above?

Reply:
I would agree. Since `fpm` is in a `try`, if it fails the next fatal error would be [here](https://github.com/thezim/PowerShell/blob/12516d8f7c6f6012c5b695ef4b2acfe89f6379de/tools/packaging/packaging.psm1#L703) leaving it in a bad state. I'll go ahead and move it.

As an aside I'm not unsure if the `try` should even be there for the `fpm` call. I would think you would want the build point of failure reported at the `fpm` call and not a dependancy later in the code.